### ISSUE TITLE
🐛 Bug Report: package novu/web:0.10.0 broken

### ISSUE THREAD

**Issue by razorness** [novuhq/novu#2413]
### 📜 Description

[ghcr.io/novuhq/novu/web:0.10.0](https://github.com/orgs/novuhq/packages/container/novu%2Fweb/60647389?tag=0.10.0) is broken which is probably caused by CLRF line endings instead of LF.

### 👟 Reproduction steps

Follow your quick start guide:
https://github.com/novuhq/novu/tree/next/docker#quick-start

### 👍 Expected behavior

A running image.

### 👎 Actual Behavior with Screenshots

Log:
```
2023-01-02 19:21:26 
2023-01-02 19:21:26 > novuhq@ start:static:web /usr/src/app
2023-01-02 19:21:26 > cross-env nx run-many --target=start:static:build --projects=@novu/web
2023-01-02 19:21:26 
2023-01-02 19:21:28 
2023-01-02 19:21:28  >  NX   Running target start:static:build for 1 project(s):
2023-01-02 19:21:28 
2023-01-02 19:21:28     - @novu/web
2023-01-02 19:21:28 
2023-01-02 19:21:28  
2023-01-02 19:21:31 
2023-01-02 19:21:31 > nx run @novu/web:"start:static:build"
2023-01-02 19:21:31 
2023-01-02 19:21:31 
2023-01-02 19:21:31 > @novu/web@0.10.0 start:static:build /usr/src/app/apps/web
2023-01-02 19:21:31 > pnpm envsetup:docker && http-server build -p 4200 --proxy http://localhost:4200?
2023-01-02 19:21:31 
2023-01-02 19:21:31 
2023-01-02 19:21:31 > @novu/web@0.10.0 envsetup:docker /usr/src/app/apps/web
2023-01-02 19:21:31 > chmod +x ./env.sh && ./env.sh && mv ./env-config.js ./build/env-config.js
2023-01-02 19:21:31 
2023-01-02 19:21:28 /bin/sh: git: not found
2023-01-02 19:21:31 /bin/sh: git: not found
2023-01-02 19:21:31 sh: ./env.sh: not found
2023-01-02 19:21:31  ELIFECYCLE  Command failed.
2023-01-02 19:21:31  ELIFECYCLE  Command failed with exit code 1.
2023-01-02 19:21:31 
2023-01-02 19:21:31  
2023-01-02 19:21:31 
2023-01-02 19:21:31  >  NX   Running target "start:static:build" failed
2023-01-02 19:21:31 
2023-01-02 19:21:31    Failed tasks:
2023-01-02 19:21:31    
2023-01-02 19:21:31    - @novu/web:start:static:build
2023-01-02 19:21:31 
2023-01-02 19:21:31    See Nx Cloud run details at https://cloud.nx.app/runs/EBXRW2RBZv
2023-01-02 19:21:31 
2023-01-02 19:21:31  ELIFECYCLE  Command failed with exit code 1.
```

See `sh: ./env.sh: not found` which is an indicator that `sh` is not able to handle CRLF.
Was able to reproduce error when setting line endings of `env.sh` to CRLF while running `docker:build`.

### 🤖 Node Version

not involved

### 📃 Provide any additional context for the Bug.

Friendly reminder to avoid CRLF where possible.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

None

**Comment by kAutoman** [comment]
Any update for this issue?
I'm facing same error.
![image](https://user-images.githubusercontent.com/98763754/210304475-8b0f2e85-de04-47c2-ad7d-464484d4bf7f.png)

**Comment by razorness** [comment]
For now, you can checkout `tag/v0.10.0`, go to folder `apps/web` and exec `pnpm docker:build` in WSL2 to have a running image to be used with docker-compose.
Image name will be `novu-web:latest`.

Ensure to have autocrfl disabled in your git config.

**Comment by scopsy** [comment]
Was also resolved in the latest 0.10.1 🙏🎉
